src/hooks/screen/useSignedChatInfoHook.ts

```diff
@@ -42,7 +43,7 @@ function useAnonymousChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
   };
 
   React.useEffect(() => {
-    initChatInfoData();
+    initializeChatInfoData();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function call in the `useEffect` hook has been updated to reflect the new function name `initializeChatInfoData`. This is consistent with the function name change.



<!-- This is an auto-generated comment by OSS CodeRabbit -->